ssv/committee.go

```diff
@@ -41,7 +41,14 @@ func (c *Committee) StartDuty(duty *types.CommitteeDuty) error {
 	if _, exists := c.Runners[duty.Slot]; exists {
 		return errors.New(fmt.Sprintf("CommitteeRunner for slot %d already exists", duty.Slot))
 	}
-	c.Runners[duty.Slot] = c.CreateRunnerFn(c.Share)
+	dutyShares := make(map[spec.ValidatorIndex]*types.Share)
+	for _, bduty := range duty.ValidatorDuties {
+		if _, exists := c.Share[bduty.ValidatorIndex]; !exists {
+			return fmt.Errorf("no share for validator %d", bduty.ValidatorIndex)
```

Comment:
shouldn't this be `continue` simply?
are we sure we want to fail the run just because of one miss?

Reply:
Yes, from the spec perspective, I would also say `continue` is better. On the other hand, in ssv, it's strange that the operator doesn't the share for the duty since he requested the duty for this validator.

Reply:
I agree